src/website/src/website-components/Logo/LogoCard.scss

```diff
@@ -0,0 +1,55 @@
+@use 'sass:map';
+@use '@ecl/theme-dev/theme';
+
+%card {
+  align-items: flex-start;
+  border: 2px solid transparent;
+  box-sizing: border-box;
+  display: flex;
+  flex-basis: 16rem;
```

Comment:
`14.5rem` to display them in 3 columns?

Reply:
@papegaill on my screen they are displayed on 4 columns, so do you mean making them bigger, in 3 columns?

Reply:
On my macbook pro screen @100%, they are displayed 2 columns
<img width="1440" alt="Screenshot 2022-01-20 at 14 22 28" src="https://user-images.githubusercontent.com/1289480/150347268-924f3428-386e-4795-8fd8-4eb4ecaf0397.png">